inspire_utils/name.py

```diff
@@ -24,13 +24,17 @@
 
 from itertools import product
 
-import six
+import logging
 from nameparser import HumanName
 from nameparser.config import Constants
+import six
 from unidecode import unidecode
```

Comment:
The import order I'm trying to informally push is
1. Future imports
2. Standard library imports
3. Vendored from the outside world
4. Vendored from Invenio
5. Vendored from INSPIRE
6. Internal/relative imports

Using the above order these imports would become
```python
# 1. Future imports
from __future__ import absolute_import, division, print_function

# 2. Standard library imports
import logging

# 3. Vendored from the outside world
import six
from nameparser import HumanName
from nameparser.config import Constants
from unidecode import unidecode
```

Perhaps I really need to add `flake8-future-import` to enforce this?

Reply:
If we're missing out on this one, then some enforcement from flake8 would be ok. 
(Btw did you mean `flake8-future-import`?)

Reply:
Whoops, no, `flake8-import-order`. Or something like that!